java/src/main/java/com/saucelabs/saucebindings/SauceOptions.java

```diff
@@ -128,6 +159,83 @@
         knownCITools.put("TeamCity", "TEAMCITY_PROJECT_NAME");
     }
 
+    public static SauceOptions chrome() {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        return chrome(chromeOptions);
+    }
+
+    public static SauceOptions firefox() {
+        FirefoxOptions firefoxOptions = new FirefoxOptions();
+        return firefox(firefoxOptions);
+    }
+
+    public static SauceOptions internetExplorer() {
+        InternetExplorerOptions internetExplorerOptions = new InternetExplorerOptions();
+        return internetExplorer(internetExplorerOptions);
+    }
+
+    public static SauceOptions edge() {
+        EdgeOptions edgeOptions = new EdgeOptions();
+        return edge(edgeOptions);
+    }
+
+    public static SauceOptions safari() {
+        SafariOptions safariOptions = new SafariOptions();
+        return safari(safariOptions);
+    }
+
+    public static SauceOptions android() {
+        AndroidOptions androidOptions = new AndroidOptions();
+        return android(androidOptions);
+    }
+
+    public static SauceOptions ios() {
+        IOSOptions iosOptions = new IOSOptions();
+        return ios(iosOptions);
+    }
+
+    public static SauceOptions chrome(ChromeOptions options) {
+        return new SauceOptions(options);
+    }
+
+    public static SauceOptions firefox(FirefoxOptions options) {
+        return new SauceOptions(options);
+    }
+
+    public static SauceOptions internetExplorer(InternetExplorerOptions options) {
+        return new SauceOptions(options);
+    }
+
+    public static SauceOptions edge(EdgeOptions options) {
+        return new SauceOptions(options);
+    }
+
+    public static SauceOptions safari(SafariOptions options) {
+        SauceOptions sauceOptions = new SauceOptions(options);
+        sauceOptions.setPlatformName(SaucePlatform.MAC_MOJAVE);
+        return sauceOptions;
+    }
+
+    public static SauceOptions android(AndroidOptions options) {
+        SauceOptions sauceOptions = new SauceOptions(options);
+        sauceOptions.setPlatformName(SaucePlatform.ANDROID);
+        sauceOptions.setDeviceName("Android GoogleAPI Emulator");
+        sauceOptions.setPlatformVersion("8.1");
```

Comment:
should it be hardcoded?

Reply:
This is a way to set a different default for each driver instead of defining one in the field directly. Honestly, I'd rather Sauce provide a default when it isn't specified, but until then this software should do it. Obviously it can be overridden as is already necessary with current code. 

Also, we probably need a way to track the things that are hard-coded that are likely to change. Ideally we can pull from a config file that can be auto-generated by the API at some point, but that is a lot of work for where we are right now.